api/src/cache.ts

```diff
@@ -52,11 +53,17 @@ function getConfig(
 	};
 
 	if (store === 'redis') {
-		const KeyvRedis = require('@keyv/redis');
-
-		config.store = new KeyvRedis(env.CACHE_REDIS || getConfigFromEnv('CACHE_REDIS_'), {
-			commandTimeout: 500,
-		});
+		const KeyvAnyRedis = require('keyv-anyredis');
+		const Redis = require('ioredis');
+		if (env.CACHE_REDIS_CLUSTER === true) {
+			const client = new Redis.Cluster(env.CACHE_REDIS || getConfigFromEnv('CACHE_REDIS_', 'CACHE_REDIS_CLUSTER'), {
+				dnsLookup: (address: any, callback: (arg0: null, arg1: any) => any) => callback(null, address),
+			});
+			config.store = new KeyvAnyRedis(client);
+		} else {
+			const client = new Redis(env.CACHE_REDIS || assign(getConfigFromEnv('CACHE_REDIS_'), { commandTimeout: 500 }));
```

Comment:
Note: our existing KeyvRedis had the option `{ commandTimeout: 500 }` and it'll use `Object.assign()` under the hood here: https://github.com/lukechilds/keyv-redis/blob/9d650f21ce4609c07d5e6848efb1572eccf95d46/src/index.js#L13

So this lodash `assign` was used to keep this option intact & resolve merge conflict.